frontend/src/APIcall.js

```diff
@@ -0,0 +1,13 @@
+// fetching the GET route from the Express server which matches the GET route from server.js
+ const getData = async () => {
+  const response = await fetch('/express_backend');
+  const body = await response.json();
+
+  if (response.status !== 200) {
+    throw Error(body.message)
+  }
+  return body;
```

Comment:
Better to wrap this with `try { } catch (error) {}` because:
1. fetch can throw error
2. `response.json()` will throw error if your response cant parse as json.